atlp-rwanda/e-comm-team-techsmith-bn
# #184797944 Implement custom database seeders script

## What does this PR do?
Implementation  of seeders in database script. 

## Description of Task to be completed?
- Creation of user, role, product, payment, cart, category and order tables/models with sequelize. 
- addition of dummy data in user table and role table.
- addition of dummy data in product table and order table.
- addition of dummy data in payment table and cart table.

## How should this be manually tested?
- Clone the repo using `git clone` repo.
- Run `npm install` from the directory root.
- Create a `.env`  to store sensitive credentials used in this application. Make sure to check `.env.example` to find all environment variables.
- Run `npm run dev` to check if database is connected.
- Run `npm run migrate` to check if migrations is running.
- Run `npm run seed` if you want to insert data  in database .

## What are the relevant pivotal tracker/Trello stories?
#184797944
